gunicorn/sock.py

```diff
@@ -166,7 +159,8 @@ def create_sockets(conf, log, fds=None):
         for fd in fds:
             sock = socket.fromfd(fd, socket.AF_UNIX, socket.SOCK_STREAM)
             sock_name = sock.getsockname()
-            sock_type = _sock_type(sock_name)
+            sockinfo = (sock.family, sock.type, sock.proto, '', socket.getsockname())
+            sock_type = _sock_type(sockinfo)
```

Comment:
Why did you pass a tuple here? `_sock_type()` only uses the first element of the tuple `sockinfo`. Can't we just pass `sock.family`?

Reply:
It was done so before. But I can change that.